src/content/docs/en/reference/modules/astro-content.mdx

```diff
@@ -137,24 +137,26 @@ A `loader` is either an object or a function that allows you to load data from a
 <Since v="6.0.0" />
 </p>
 
-`schema` is an optional Zod object to configure the type and shape of document frontmatter for a live collection. Each value must use [a Zod validator](https://github.com/colinhacks/zod).
+An optional Zod object to configure the type and shape of your data for a live collection. Each value must use [a Zod validator](https://github.com/colinhacks/zod).
```

Comment:
Question: is the `schema`  property any different for the two collections? I'm wondering whether these should just be combined into one if not?

(Then, the line can say e.g. "When you define a schema in a live collection, it will take precedence over the live loader’s types when you query the collection."